body.tex

```diff
@@ -0,0 +1,481 @@
+
+In a special case, that an issue cannot be fixed on master, the ticket branch can be opened based on the release branch.
+
+The porting can be applied also in case that an issue, implemented on master after the first release candidate has been created, has to be included in the release.
+
+The DMCCB has to overview the backporting process and take corrective actions if needed. 
+Backporting may require considerable use of development resources and/or delay in the final release availability.
+
+
+\subsection{Additional Release Candidates} \label{sec:newrc}
+
+Once one or more issues have been fixed on the release branch, a new release candidate has to be generated.
+The same steps used for generating the first release candidate, need to be used also in this case, with different parameters:
+
+\begin{itemize}
+\item GIT\_REFs: the starting Git references, for example "b17.0.x 17.0.0.rc1". 
+This implies, where available the release branch will be used instead of the previous release candidate
+\item GIT\_TAG: the release candidate to be generated, for example "17.0.0.rc2"
+\end{itemize}
+
+Further release candidates can be created just repeating the process with the appropriate parameters.
+The release candidate 3 will require the following parameters:
+
+\begin{itemize}
+\item GIT\_REFs: the starting Git references, for example "b17.0.x 17.0.0.rc2". 
+\item GIT\_TAG: the release candidate to be generated, for example "17.0.0.rc3"
+\end{itemize}
+
+Generalizing, a release candidate N will be based on the release branch and the previous release candidate N-1.
+
+
+\subsection{Final Release} \label{sec:finalrelease}
+
+Once a final release candidate \textbf{Nf} has been identified, the final release can be created.
+
+To create the final release, just repeat the same steps used to create the release candidates, using the following parameters:
+
+\begin{itemize}
+\item GIT\_REFs: "17.0.0.rcNf"
+\item GIT\_TAG: "17.0.0"
+\end{itemize}
+
+Note that the final release is from a point of view of content identical to the last release candidate.
+
+Final tags need to be done manually for all the additional documentation, environments and other packages.
+
+\newpage
+
+\section{Patch Releases} \label{sec:patchreleases}
+
+In the case that the DMCCB approves a proposed patch release, the process shall be:
+
+\begin{itemize}
+\item create the release branches from the available release tags on the Github packages impacted by the fixes (if they don't exist)
+\item backport the requested issues
+\item create a release candidate, for example \textit{17.0.1.rc1} using the the same approach explained above (\ref{sec:newrc})
+\item validate the release candidate
+\item fix eventual problems found in the release candidate and repeat the last 2 steps, create a new release candidate a validate it, until a valid release candidate is found
+\item create a final release as described above (\ref{sec:finalrelease}).
```

Comment:
And presumably, have some additional standards like "No API Changes, etc..."

Reply:
Right, but that should be part of the DMCCB approval of the patch.
In that case, a `regular` release should be done from master.

I am not sure this is the right place where to say this.
Maybe the 3.6 subsection in LDM-294 needs to be expanded a little bit.